docs/src/code/docs-style-guide.adoc

```diff
@@ -411,157 +493,196 @@ Follow these guidelines for punctuation:
 * End full sentences with a period.
 * Use one space between sentences.
 * Do not use semicolons. Use two sentences instead.
-* Do not use double spaces. (Tested in https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/.vale/gitlab/SentenceSpacing.yml[`SentenceSpacing.yml`].)
-* Do not use non-breaking spaces. Use standard spaces instead. (Tested in https://gitlab.com/gitlab-org/gitlab/-/blob/master/scripts/lint-doc.sh[`lint-doc.sh`].)
-* Do not use tabs for indentation. Use spaces instead. You can configure your code editor to output spaces instead of tabs when pressing the tab key.
-* Use serial (Oxford) commas before the final *and* or *or* in a list of three or more items. (Tested in https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/.vale/gitlab/OxfordComma.yml[`OxfordComma.yml`].)
+* Do not use double spaces.
+// TODO Test with https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/.vale/gitlab/SentenceSpacing.yml[`SentenceSpacing.yml`].
+* Do not use non-breaking spaces. Use standard spaces instead.
+// TODO Test with https://gitlab.com/gitlab-org/gitlab/-/blob/master/scripts/lint-doc.sh[`lint-doc.sh`].
+* Do not use tabs for indentation. Use spaces instead. +
+  You can configure your code editor to output spaces instead of tabs
+  when pressing the tab key.
+* Use serial (Oxford) commas before the final *and* or *or* in a list of
+  three or more items.
+// TODO Test with https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/.vale/gitlab/OxfordComma.yml[`OxfordComma.yml`].
 * Avoid dashes. Use separate sentences, or commas, instead.
```

Comment:
This might be a code place to mention that HAL pin names require a "minus" sign rather than em-dash or similar if users are to be able to paste code samples into their HAL files without getting very confusing errors.